src/main/java/com/blackduck/integration/blackduck/imageinspector/image/oci/OciImageDirectoryExtractor.java

```diff
@@ -140,36 +169,59 @@ private ArchiveFileType parseArchiveTypeFromLayerDescriptorMediaType(String medi
         } else if (mediaType.endsWith(LAYER_ARCHIVE_TAR_ZSTD_MEDIA_TYPE_SUFFIX)) {
             return ArchiveFileType.TAR_ZSTD;
         } else {
-            throw new IntegrationException(String.format("Unrecognized layer media type: %s", mediaType));
+            throw new IntegrationException(String.format("Possible unsupported input archive file type. Please refer to Docker Inspector documentation [link]. Unrecognized media type %s of layer %s.", mediaType, digest));
```

Comment:
@cpottsbd What should be the link to the documentation of supported OSs?

Reply:
OS support is not precise. (This is what I have gleaned from some support conversations.)
That said, I will endeavor to update the information on this page, (So link to this page please.) https://documentation.blackduck.com/bundle/detect/page/packagemgrs/docker/formats.html